website/docs/r/team_token.html.markdown

```diff
@@ -32,6 +32,20 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) A date and time in which the team token will expire. The expiration date must be passed in
+iso8601 format. If no expiration date is supplied, the expiration date will default to null and never expire.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_team" "test" {
```

Comment:
I think that this is a copy-paste error, this looks like the configuration for a team, not a team token

Reply:
Great catch! I've updated this.